client/layout/guided-tours/tours/boosted-post-tour.js

```diff
@@ -30,11 +31,12 @@ export const BoostedPostTour = makeTour(
 		when={ and(
 			isEnabled( 'boosted-post-survey' ),
 			isDesktop,
-			isAbTestInVariant( 'boostedPostSurvey', 'enabled' )
+			isAbTestInVariant( 'boostedPostSurvey', 'enabled' ),
+			isSelectedSitePlanPaid,
 		) }
 	>
 		<Step name="init" placement="right">
-			<p>{ translate( 'Wouldn\'t it be nice if your posts reached a wider audience?' ) }</p>
+			<p>{ translate( 'Want to reach a bigger audience? Take our survey and help shape a new feature.' ) }</p>
```

Comment:
We should use bold for the first sentence for consistency with other tours — see [example](https://github.com/Automattic/wp-calypso/blob/75bd34b8b42e492499c0cadc223641495288d545/client/layout/guided-tours/tours/main-tour.js#L44).